libs/wire-api/src/Wire/API/Routes/Public/Brig.hs

```diff
@@ -469,6 +484,21 @@ type UserHandleAPI =
                     '[Respond 200 "Handle is taken" ()]
                     ()
            )
+    :<|> Named
+           "check-user-handle"
+           ( Summary "Check whether a user handle can be taken"
+               :> From V7
+               :> CanThrow 'InvalidHandle
+               :> CanThrow 'HandleNotFound
+               :> ZUser
+               :> "handles"
```

Comment:
this will require nginz to be changed. what about changing `/:uid/` into `/uid/:uid/`, so the old `"check-user-handle"` can stay?